docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -160,6 +160,18 @@ This is a known problem when using the `jsdom` library (a dependency of Jest) wi
 
 Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. Please see the Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages) on how you could achieve this.
 
+### Next 13.4+: `Module not found errors`
+When you use Connected components, you may see the follow errors from `aws-crt` and `encoding` on the server:
+
+```bash
```

Comment:
Nit: use unstyled codeblock instead of bash codeblock so that `'s` don't mess up the syntax highlighting:

![image](https://github.com/aws-amplify/amplify-ui/assets/43682783/b2b7ef6e-35ea-498e-97ee-62a825ad6d55)